papers.html

```diff
@@ -136,6 +136,27 @@
+      <p></p>
+      <h5>Andrews et al. 2024</h5>
+      <p></p>
+
+      <p>
+        <button class="btn btn-secondary" type="button" data-toggle="collapse" data-target="#Andrews20241104" aria-expanded="false" aria-controls="Andrews20241104">
+          Details
+        </button>
+      </p>
+        <div class="collapse" id="Andrews20241104">
+          <div class="card card-body">
+            <p><b>Authors: </b> Jeff J. Andrews, Simone S. Bavera, Max Briel, Abhishek Chattaraj, Aaron Dotter, Tassos Fragos, Monica Gallegos-Garcia, Seth Gossage, Vicky Kalogera, Eirini Kasdagli, Aggelos Katsaggelos, Chase Kimball, Konstantinos Kovlakas, Matthias U. Kruckow, Camille Liotine, Devina Misra, Kyle A. Rocha, Dimitris Souropanis, Phillip M. Srivastava, Philipp M. Srivastava, Meng Sun, Elizabeth Teng, Zepei Xing, Emmanouil Zapartas, Michael Zevin</p>
+            <p><b>Access: </b> <a href="https://arxiv.org/abs/2411.02376" target="_blank">arXiv:2411.02376</a> || <a href="https://ui.adsabs.harvard.edu/abs/arXiv:2411.02376" target="_blank">NASA ADS</a></p>
+            <p><b>Abstract: </b>Whether considering rare astrophysical events on cosmological scales or unresolved stellar populations, accurate models must account for the integrated contribution from the entire history of star formation upon which that population is built. Here, we describe the second version of \posydon, an open-source binary population synthesis code based on extensive grids of detailed binary evolution models computed using the \mesa code, which follows both stars' structures as a binary system evolves through its complete evolution from the zero-age main sequence, through multiple phases of mass transfer and supernovae, to their death as compact objects. To generate synthetic binary populations, \posydon uses advanced methods to interpolate between our large, densely spaced grids of simulated binaries. In our updated version of \posydon, we account for the evolution of stellar binaries across a cosmological range of metallicities, extending from $10^{-4}\,\Zsun$ to $2\,\Zsun$, including grids specifically focused on the Small and Large Magellanic Clouds ($0.2\,\Zsun$ and $0.45\,\Zsun$). In addition to describing our model grids and detailing our methodology, we outline several improvements to \posydon. These include the incorporation of single stars in stellar populations, a treatment for stellar mergers, and a careful modeling of ``reverse-mass transferring" binaries, in which an once-accreting star later becomes a donor star. Our simulations are focused on binaries with at least one high-mass component, such as those that host neutron stars and black holes, and we provide post-processing methods to account for the cosmological evolution of metallicity and star formation as well as rate calculations for gravitational wave events, gamma-ray bursts, and other transients. </p>
```

Comment:
Please replace LaTeX commands which are not type set on html pages.

Reply:
```suggestion
            <p><b>Abstract: </b>Whether considering rare astrophysical events on cosmological scales or unresolved stellar populations, accurate models must account for the integrated contribution from the entire history of star formation upon which that population is built. Here, we describe the second version of POSYDON, an open-source binary population synthesis code based on extensive grids of detailed binary evolution models computed using the \mesa code, which follows both stars' structures as a binary system evolves through its complete evolution from the zero-age main sequence, through multiple phases of mass transfer and supernovae, to their death as compact objects. To generate synthetic binary populations, POSYDON uses advanced methods to interpolate between our large, densely spaced grids of simulated binaries. In our updated version of POSYDON, we account for the evolution of stellar binaries across a cosmological range of metallicities, extending from 10<sup>-4</sup>&nbsp;<i>Z</i><sub>&odot;</sub> to 2&nbsp;<i>Z</i><sub>&odot;</sub>, including grids specifically focused on the Small and Large Magellanic Clouds (0.2<i>Z</i><sub>&odot;</sub> and 0.45&nbsp;<i>Z</i><sub>&odot;</sub>). In addition to describing our model grids and detailing our methodology, we outline several improvements to POSYDON. These include the incorporation of single stars in stellar populations, a treatment for stellar mergers, and a careful modeling of "reverse-mass transferring" binaries, in which an once-accreting star later becomes a donor star. Our simulations are focused on binaries with at least one high-mass component, such as those that host neutron stars and black holes, and we provide post-processing methods to account for the cosmological evolution of metallicity and star formation as well as rate calculations for gravitational wave events, gamma-ray bursts, and other transients. </p>
```

Reply:
thanks, updated